cmd/buildah/delete.go

```diff
@@ -10,20 +10,21 @@ var (
 	deleteFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "name",
-			Usage: "name or ID of the working container",
+			Usage: "name or `ID` of the working container",
 		},
 		cli.StringFlag{
 			Name:  "root",
-			Usage: "root directory of the working container",
+			Usage: "root `directory` of the working container",
```

Comment:
I really do think that leaving just "directory" marked, so that the --help output for the flag is
   --root directory   root directory of the working container
better than changing it, which would produce
   --root root directory  root directory of the working container
is equally informative.